update 3DGS-DET ( 3DGS for 3D Object Detection)

yangcaoai opened the pull request.

MrNeRF commented on README.md:
```diff
@@ -394,8 +395,20 @@ rendering on several established datasets.
   [📄 Paper (Low Resolution)](https://repo-sam.inria.fr/fungraph/3d-gaussian-splatting/3d_gaussian_splatting_low.pdf) | [📄 Paper (High Resolution)](https://repo-sam.inria.fr/fungraph/3d-gaussian-splatting/3d_gaussian_splatting_high.pdf) | [🌐 Project Page](https://repo-sam.inria.fr/fungraph/3d-gaussian-splatting/) | [💻 Code](https://github.com/graphdeco-inria/gaussian-splatting) | [🎥 Short Presenta ...
 
-<br>
+<be>
```
Typo

MrNeRF commented on README.md:
```diff
@@ -394,8 +395,20 @@ rendering on several established datasets.
+
+  [📄 Paper](https://arxiv.org/pdf/2410.01647) | [💻 Code ](https://github.com/yangcaoai/3DGS-DET) 
+
+  
```
<br>

MrNeRF commented:
Thanks for the contribution. Please check the review. Additionally, as the code is not released, please mark it as done for the other papers.

yangcaoai commented:
> **MrNeRF ** requested changes
> ### The MrNeRF:main branch requires linear history
> The base branch does not accept merge commits. Alternate merge methods are preferred. [Learn more about required linear history.](https://docs.github.com/github/administering-a-repository/requiring-a-linear-commit-history)

Hi, I've just updated the existing pull request with the requested changes. I'm not very familiar with the pull request mechanism, so could you please let me know if I've addressed all of your comments correctly? Thank you for your guidance!

MrNeRF commented:
All good. Thx!

yangcaoai commented:
> All good. Thx!

Thank you for your great contribution to the community :)
